compiler/src/typed/oprint.re

```diff
@@ -738,8 +766,8 @@ and print_out_type_decl = (kwd, ppf, td) => {
     | Otyp_sum(constrs) =>
       fprintf(
         ppf,
-        " =@;<1 2>%a",
-        print_list(print_out_constr, ppf => fprintf(ppf, "@ | ")),
+        " {\n@;<2 2>%a\n}",
```

Comment:
I think those newlines could give us problems down the line... Curious to see if this would do the trick here:
```suggestion
        "@[<v> {@;<2 2>%a@;}@]",
```

Reply:
@ospencer these changes resulted in this:
```
enum Range {
            Inclusive(Number, Number),
            Exclusive(Number, Number),
          }
```
Any ideas?

Reply:
I tweaked this against graindoc and pushed it up. Take a look!